maven-xml/src/main/java/org/apache/maven/xml/SAXEventUtils.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.maven.xml;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Utility class for SAXEvents
+ * 
+ * @author Robert Scholte
+ * @since 4.0.0
+ */
+public final class SAXEventUtils
+{
+    private SAXEventUtils()
+    {
+    }
+    
+    public static String renameQName( String oldQName, String newLocalName )
+    {
+        return oldQName.replaceFirst( "[^:]+$", newLocalName );
```

Comment:
Precompile the pattern since it is reused?

Reply:
Sure, that's a simple change

Reply:
Fixed